### ISSUE TITLE
Black Screen on MacOS

### ISSUE THREAD

**Issue by JoeDog1** [minecraft-linux/mcpelauncher-manifest#996]
**Describe the bug**
Whenever I open Bedrock Launcher any Version past 1.18.12.01 the launcher opens, but the Screen is black

**To Reproduce**
1. Go to open latest version of minecraft on MacOs
2. See error 

**Expected behavior**
The screen should appear normal

**Screenshots**

![Screen Shot ](https://github.com/minecraft-linux/mcpelauncher-manifest/assets/114356390/6f53a993-5c89-45ff-a2ae-37f3d32cd94b)

![Screen Shot ](https://github.com/minecraft-linux/mcpelauncher-manifest/assets/114356390/a5cedfe7-c6d0-44f0-9dc8-693386455ef9)

**Desktop:**

 - OS: MacOS
 - Version: 1.20.15.01 (x86_64)
 - Installation Type: dmg

**Additional context**
This case seems very similar to https://github.com/minecraft-linux/mcpelauncher-manifest/issues/834
But when I tried the [MESA_EXTENSION_OVERRIDE=-GL_EXT_instanced_arrays] override It wouldn't work. Also it says later in the Github that the dev.'s "fixed" it in the source code. My suspicion is that in this case it is for Linux and not MacOs and that this problem was only fixed for Linux.

**Comment by ChristopherHX** [comment]
Sorry, but I don't think this problem can be fixed due to the age of your GPU.

macbooks from 2012 onwards have almost (can't be shure about this, no everyone provides their model year) no such problems

Your old GPU might not have all required undocumented GLEXT, because you OpenGL ES level is 2.

If an opengl es 2/3 feature is not available bgfx will show a black screen.

Mesa related bugs have nothing todo with macOS, macOS had much longer always the blackscreen as result that delayed renderdragon support for months.

Due to the age of your Hardware, you probably have no metal 2.1 and you can skip testing metal.

**Comment by ChristopherHX** [comment]
Software rendering has been a way, but 1-3 fps isn't worth it to try

**Comment by ChristopherHX** [comment]
I might create a blackscreen/crash detection and relaunch automatically via software rendering at cost of performance

**Comment by JoeDog1** [comment]
So is the only way to fix this to update my computer??

**Comment by JoeDog1** [comment]
because

> I might create a blackscreen/crash detection and relaunch automatically via software rendering at cost of performance

Because I don't know how to do this

**Comment by ChristopherHX** [comment]
> So is the only way to fix this to update my computer??

No, **replacing** your computer would be necessary with a model from 2012 or later.

> Because I don't know how to do this

You can't play a game with 1-3 frames per second (fps) (normally 60 frames per second), the game won't respond to your mouse and extreme stutter

That would only resolve the black screen, but doesn't make the game playable
